src/args.ts

```diff
@@ -84,6 +109,7 @@ function getParams() {
     .example('$0 deploy -c config.json -i path/to/files', 'Deploy Auth0 via Path')
     .epilogue('See README (https://github.com/auth0/auth0-deploy-cli) for more in-depth information on configuration and setup.')
     .wrap(null);
+  //@ts-ignore because we know these types are either ExportParams or ImportParams. TODO: fix more native way of inferring types from yargs
```

Comment:
It's a real shame that `yargs` doesn't provide a simpler, more native way of inferring types. I can assure with certainty that the types I've set in the function signature are correct and the ones auto-generated are wrong.